core/domain/topic_domain.py

```diff
@@ -373,7 +411,9 @@ def to_dict(self):
         }
 
     @classmethod
-    def from_dict(cls, subtopic_dict):
+    def from_dict(
+        cls, subtopic_dict: SubtopicDict
+    ) -> Subtopic:
```

Comment:
```suggestion
    def from_dict(cls, subtopic_dict: SubtopicDict) -> Subtopic:
```

ditto elsewhere, if it can fit one line just put it on one line

Reply:
done.